modules/installation-configuration-parameters.adoc

```diff
@@ -313,6 +319,15 @@ ifdef::osp[]
    }
 }
 ----
+
+|`platform.openstack.machinesSubnet`
+|The UUID of a {rh-openstack} subnet that the cluster's nodes use.
+//Where are these params?
+The first CIDR in `networks.machineNetwork` must match the CIDR of `machinesSubnet`.
```

Comment:
@iamemilio Where are these params defined?

Reply:
Oh, is it "networking" or "networks?"

Reply:
The CIDR of a subnet is defined when you create it. It can be looked up with the command: openstack subnet show <id>

networks.machineNetwork is a parameter in the networks object in the install-config.yaml

Reply:
@iamemilio Sorry, got confused there. Did the `networking` object become `networks` or is that a typo?

Reply:
its the section that says networking here: https://github.com/openshift/installer/blob/master/docs/user/customization.md#platform-customization

Reply:
Thanks! Just wanted to make sure the naming was correct + consistent.